packages/node_modules/@webex/webex-core/src/config.js

```diff
@@ -65,7 +65,7 @@ export default {
       hydra: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1',
       hydraServiceUrl: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1'
     },
-    validateDomains: process.env.VALIDATE_DOMAINS === 'true' || false,
+    validateDomains: true,
```

Comment:
Don't need this one, see [this case](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-135715).

Reply:
The line can be removed completely?

Reply:
Yes, as all of these values are no longer used by the device plugin. There's a task to remove this information in our backlog.

Reply:
Can you link the task?